finetuner-workflow/finetuner/Dockerfile

```diff
@@ -1,14 +1,15 @@
 # syntax=docker/dockerfile:1.2
 
-ARG BASE_IMAGE=ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda11.8.0-torch2.0.0-vision0.15.1
+ARG BASE_IMAGE=ghcr.io/coreweave/ml-containers/torch:ef85248-base-cuda11.8.0-ubuntu20.04-torch2.3.0-vision0.18.0-audio2.3.0
 
 # Dependencies requiring NVCC are built ahead of time in a separate stage
 # so that the ~2 GiB dev library installations don't have to be included
 # in the final finetuner image.
 # gcc-10/g++-10/lld do not need to be installed here, but they improve the build.
 # gfortran-10 is just for compiler_wrapper.f95.
 FROM ${BASE_IMAGE} as builder
-RUN apt-get install -y --no-install-recommends \
+RUN  apt-get update -y && \
+    apt-get install -y --no-install-recommends \
         cuda-nvcc-11-8 cuda-nvml-dev-11-8 libcurand-dev-11-8 \
         libcublas-dev-11-8 libcusparse-dev-11-8 \
         libcusolver-dev-11-8 cuda-nvprof-11-8 \
```

Comment:
Probably not needed with our `torch` base images:
- `libcurand`
- `libcublas`
- `libcusparse`
- `libcusolver`

If you are using `torch-extras` then `cuda-nvcc` and `cuda-nvprof` (and most likely `cuda-nvml` unless we use it for other reasons than for building libraries) are probably all also not needed.